src/string_bytes.cc

```diff
@@ -568,18 +590,57 @@ static void force_ascii(const char* src, char* dst, size_t len) {
   }
 }
 
+#define packed(byte) (static_cast<uint8_t>(byte) * 0x01010101u)
```

Comment:
Can you make this an inline (possibly constexpr) function?  Rule of thumb: macros when you must, functions when you can.

Name: `pack()` instead of `packed()`?  The former implies action, the latter sounds more like a predicate.